test/metabase/actions_test.clj

```diff
@@ -560,3 +560,70 @@
                           (is (some (partial instance? org.apache.sshd.common.SshException)
                                     (u/full-exception-chain response))
                               "None of the errors are from ssh"))))))))))))))
+
+(defmacro with-uuids-test-data-and-actions-permissively-enabled!
+  "Combines [[mt/with-actions-test-data-and-actions-enabled]] with full permissions."
+  {:style/indent 0}
+  [& body]
+  `(mt/with-temp-test-data [["people"
+                             [{:field-name "id"
+                               :base-type :type/UUID
+                               :pk? true}
+                              {:field-name "name"
+                               :base-type :type/Text}]
+                             [[#uuid "d6b02fa2-bf7b-4b32-80d5-060b649c9859" "Tim"]
+                              [#uuid "d39bbe77-4e2e-4b7b-8565-cce90c25c99b" "Bob"]
+                              [#uuid "b34b5c6e-2613-494a-9278-8a74c7ea915e" "Jane"]]]]
+     (mt/with-actions-enabled
+       (binding [*current-user-permissions-set* (delay #{"/"})]
+         ~@body))))
+
```

Comment:
Using an ad-hoc dataset like this, but whose name coincides with one of the Sample Database tables, is a trap for the unwary reader of the code.

Can we rename this table to `uuid_people` or something, or just make up a totally new table name like `employees` or `cafeteria_staff`?